provers/groth16/tests/groth16.rs

```diff
@@ -0,0 +1,35 @@
+use lambdaworks_groth16::{common::*, qap_example_circuit_1, setup, verify, Proof};
+
+#[test]
```

Comment:
It would be awesome to have tests that cross validate intermediate results with other libraries like gnark.